src/main/shop/service/ShopFileReaderImpl.java

```diff
@@ -0,0 +1,25 @@
+package service;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ShopFileReaderImpl implements ShopFileReader {
+    @Override
+    public List<String> readFromFile(String fileName) {
+        List<String> readFile = new ArrayList<>();
+        try (BufferedReader reader = new BufferedReader(new FileReader(fileName))) {
+            String line = reader.readLine();
```

Comment:
we should not skip the first line here - we should be able to reuse this implementation for reading from another files